### PR TITLE
add Erp e2e tests

### PR DESCRIPTION
**Pull request by KrupaPammi**

### CODE REVIEW

**Review by lebaz20 — COMMENTED** [review]
Good job, just minor comments.

File: tests/commands/form.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
+const ChanceJS = require('chance');
+
+const chance = new ChanceJS();
+
+const randomString = chance.string({
```
[the remark this answers is not in the document]
Comment (KrupaPammi) [reply tests/commands/form.js]: realised it's not needed, so have removed it now

File: tests/commands/form.js [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+      .setValue('#field-input--town', 'London')
+      .click('button[type=submit]')
+      .pause(5000);
+  },
+
+  /**
+   * Populate the giftaid using postcode lookup
+   * @param client
+   */
+  fillFormPrefilledMobile: function (client) {
```
[the remark this answers is not in the document]
Comment (KrupaPammi) [reply tests/commands/form.js:42]: fillFormPrefilledMobile: is for this test case where the url loads with a token which has a prefilled number. `fillform` command has a mobile number to type

```'User completes main giftaid journey from link with token': function (client) {
     client.url(process.env.BASE_URL + 'uX8R5SzcKfk=').maximizeWindow().waitForElementVisible('body', 1000);
     client.page.mainGA().fillFormPrefilledMobile(client);
     client.page.form().fillFormPrefilledMobile(client);
     client.waitForElementVisible('div > h1', 2000);
     client.expect.element('div > h1').text.to.equal('Thank you,\n' +
       'test!');
     client.end();
   },
};```

File: package.json [hunk 3]
```diff
@@ -30,13 +30,15 @@
     "react-router-dom": "^4.2.2",
     "snyk": "^1.316.1",
     "susy": "^3.0.3",
-    "whatwg-fetch": "2.0.3"
+    "whatwg-fetch": "2.0.3",
+    "uuid": "latest"
```
[the remark this answers is not in the document]
Comment (KrupaPammi) [reply package.json]: have changed it to `"uuid": "8.3.2"`

**Review by lebaz20 — APPROVED** [review]

**Comment by lebaz20** [comment]
I did some changes to tests, if it's fine with you, then feel free to merge it.
I added a nightly test job via https://github.com/comicrelief/serverless-giftaid-bootstrap/pull/3 as well.

**Comment by KrupaPammi** [comment]
Thanks @lebaz20

**Pull request merged**